edx/blockstore
# FAL-2522 Remove period from python version makefile var

[FAL-2522](https://tasks.opencraft.com/browse/FAL-2522)

Using a period in the docker-compose project name has been observed to cause issues, when using docker-compose 2.0.
For example: https://github.com/edx/blockstore/pull/128#pullrequestreview-774505053
So, normalize all references to 3.8 to 38.
This fixes the issue, and still retains the filenaming linked with the python version used.

**Test instructions**:

- install docker-compose 2.0
- run `make easyserver` and verify it is brought up successfully
- run `make destroy` and verify it is destroyed as expected
- run `make testserver` and verify the testserver is brought up successfully

**Author notes/concerns**:

What's stopping us from removing all these superfluous variables (`PYTHON_VERSION`, `BLOCKSTORE_*`) and renaming the files to their defaults (`Dockerfile`, `docker-compose.yml`)?  The makefile seems very overengineered for what is essentially a convenience wrapper around docker-compose.
